ja/faq/netlify-deployment.md

```diff
@@ -23,10 +23,25 @@ Netlify ダッシュボードの _"New site from Git"_ ボタンを押下しま
 
 ## 操作：
 
+### 静的に生成されたサイトの場合
+
 1. __ブランチをデプロイする：__ `master` もしくはデプロイしたいブランチ
 2. __ビルドコマンド：__ `npm run generate`
 3. __公開ディレクトリ：__ `dist`
 
+
+## SPAモードで生成されたサイトの場合
```

Comment:
```suggestion
## SPA モードで生成されたサイトの場合
```

Please put a space between half-width characters and full-width characters. :pray: